modules/iam-primary-roles/cicd-policy.tf

```diff
@@ -7,7 +7,7 @@ data "aws_iam_policy_document" "cicd" {
       "sts:TagSession",
     ]
     resources = [
-      "arn:aws:iam::*:role/*-helm",
+      format("arn:%s:iam::*:role/*-helm", data.aws_partition.current.partition),
```

Comment:
@Gowiem should I use the `aws_caller_identity` data source in place of the `*` for the account id?

Reply:
That's a good question for @nitrocode or others -- I haven't actually used the account-map and similar roles in my work, so I'm not sure if that is deliberately open or not. Seems like a security hole to me...

Reply:
wondering if it would also make sense to replace references to `var.primary_account_id` with data from `aws_caller_identity`

Reply:
@mikedizon my assumption would be that `var.primary_account_id` was added for the purpose of enabling cross account permissioning so that wouldn't be a good call. Add nitro so he can give this a set of eyes and provide more detail on this component.